Revamp Iteration realted BBEs (#3910)
ballerina-platform/ballerina-distribution: examples/while-statement/while_statement.md

# While statement

The `while` statement can be used to repeat a block of statements until a boolean condition is true.

The parentheses ( ) around the boolean expression are not required as they are in other languages. But it is required to enclose the statement block using curly braces { }.

::: code while_statement.bal :::

> 

Reviewer: Add related Links, continue, while, foreach.

# While statement

> The `while` statement can be used to repeat a block of statements until a boolean condition is true. The parentheses `( )` around the boolean expression are not required as they are in other languages. But it is required to enclose the statement block using curly braces `{ }`.

Reviewer 2: ```suggestion
The `while` statement can be used to repeat a block of statements until a boolean condition is true. The parentheses `( )` around the boolean expression are not required. However, it is required to enclose the statement block using curly braces `{ }`.
```

# While statement

> The `while` statement can be used to repeat a block of statements until a boolean condition is true.
> 
> The parentheses ( ) around the boolean expression are not required as they are in other languages. But it is required to enclose the statement block using curly braces { }.

Reviewer 3: ```suggestion
The `while` statement can be used to repeat a block of statements until a boolean condition is true. The parentheses `( )` around the boolean expression are not required as they are in other languages. But it is required to enclose the statement block using curly braces `{ }`.
```

::: code while_statement.bal :::

::: out while_statement.out :::

## Related links
- [Continue statement](/learn/by-example/continue-statement/)

> - [Foreach statement](/learn/by-example/foreach-statement/)

Reviewer 3: Shall we have break and continue BBEs and then foreach BBE?
